include/zephyr/bluetooth/mesh/dfd_srv.h

```diff
@@ -43,6 +75,7 @@ struct bt_mesh_dfd_srv;
 			.blob = { .cb = &_bt_mesh_dfd_srv_blob_cb },           \
 		},                                                             \
 	}
+#endif /* CONFIG_BT_MESH_DFD_SRV_OOB_UPLOAD */
```

Comment:
Can we keep both macro when `CONFIG_BT_MESH_DFD_SRV_OOB_UPLOAD` is enabled? This will allow to compile the dfd server code that didn't use OOB before.